sindresorhus/refined-github
# Restore `view-markdown-source` buttons

Closes #2619 



# Test

- Go to some markdown file (i.e. https://github.com/sindresorhus/refined-github/blob/master/contributing.md)
--> Notice the two buttons to display the source and rendered markdown are displayed (1st set of buttons on the screenshot just below)
<img width="431" alt="Screenshot 2019-12-12 22 39 00" src="https://user-images.githubusercontent.com/1215056/70751274-4e72bf00-1d30-11ea-97bb-3a20388dc333.png">
- Click on the first button
--> This should show the source of the markdown file
- Click on the second button
--> This should show the rendered markdown file
